react-frontend/src/pages/AddListing.js

```diff
@@ -207,6 +209,34 @@ function AddListing() {
             value={listing.title}
             onChange={handleChange}
           />
+          <label htmlFor="category">Category</label>
+          <select
+          id="category"
+          name="category"
+          onChange={handleChange}>
+            <option value="Other">Other</option>
+            <option value="Vehicles">Vehicles</option>
+            <option value="Property Rentals">Property Rentals</option>
+            <option value="Apparel">Apparel</option>
+            <option value="Classifieds">Classifieds</option>
+            <option value="Electronics">Electronics</option>
+            <option value="Entertainment">Entertainment</option>
+            <option value="Family">Family</option>
+            <option value="Free Stuff">Free Stuff</option>
+            <option value="Garden & Outdoor">Garden & Outdoor</option>
+            <option value="Hobbies">Hobbies</option>
+            <option value="Home Goods">Home Goods</option>
+            <option value="Home Improvement">Home Improvement</option>
+            <option value="Supplies">Supplies</option>
+            <option value="Home Improvement Supplies">Home Improvement Supplies</option>
+            <option value="Home Sales">Home Sales</option>
+            <option value="Musical Instruments">Musical Instruments</option>
+            <option value="Office Supplies">Office Supplies</option>
+            <option value="Pet Supplies">Pet Supplies</option>
+            <option value="Sporting Goods">Sporting Goods</option>
+            <option value="Toys & Games">Toys & Games</option>
+            <option value="Buy and Sell Groups">Buy and Sell Groups</option>
+          </select>
```

Comment:
**Like my previous comment, I would recommend declaring a categories array and using a lambda mapping function:**

`const categories = [
  "No Change",
  "Other",
  "Vehicles",
  etc...
];`

`{categories.map((category, index) => (
            <option key={index} value={category}>
              {category}
            </option>
          ))}`

Reply:
+1

Reply:
I would also say, perhaps you can make categories a component so you can update the options in one spot instead and if we need to add categories to any other part of the code we don't need to copy the whole array over.